get_glottolog_language_table.R

```diff
@@ -0,0 +1,26 @@
+#This script takes the values and languages tables from a cldf-release and combines then and transforms them to a wide data format from a long. It does not take into account the parameter or code tables.
+
+source("requirements.R")
+
+if(!file.exists("output/processed_data/glottolog_4.3_languages.tsv")){
```

Comment:
You should use the same glottolog version as we used for everything else. Does ./results/Glottolog_Languages.csv have what is needed?